aggregator/src/test/java/com/redhat/cloud/notifications/DailyEmailAggregationJobTest.java

```diff
@@ -136,7 +138,7 @@ void shouldNotIncreaseAggregationsWhenHostIdIsDifferent() {
     void shouldNotProcessMailsWhenNewCronJobIsDisabledByDefault() {
         final EmailAggregationResources emailAggregationResources = mock(EmailAggregationResources.class);
         final ObjectMapper objectMapper = mock(ObjectMapper.class);
-        DailyEmailAggregationJob dailyEmailAggregationJob = new DailyEmailAggregationJob(emailAggregationResources, objectMapper);
+        DailyEmailAggregationJob dailyEmailAggregationJob = new DailyEmailAggregationJob(emailAggregationResources, objectMapper, new SimpleMeterRegistry());
```

Comment:
Shouldn't the counter value be checked in all tests?

Reply:
I actually don't want to test the gauge in that state. Will probably have to add another gauge to visualize more things and will then aggregate all gauges containing to the cron job in the same custom gauge. This is just a first step.